README.md

```diff
@@ -25,6 +25,15 @@ Bluejay aims to be an open source successor to BLHeli_S adding several improveme
 
 See the project [changelog](CHANGELOG.md) for a list of changes.
 
+## Supported Platforms
+- SiLabs EFM8 BB21
+- SiLabs EFM8 BB51
+
+### Deprecated Platforms
+The `SiLabs EFM8 BB10` has been deprecated. Thas last official release is [0.18.1](https://github.com/bird-sanctuary/bluejay/releases/tag/v0.18.1). The BB10 has it's limitations in comparison to the BB21 and BB51. Not many of those seem to be around anymore and the team does not have a single one for testing, thus decided to drop support.
```

Comment:
Thas last official release... -> The last official release...

Reply:
Fixed.